chat-sample/src/extension.ts

```diff
@@ -155,6 +166,19 @@ export function activate(context: vscode.ExtensionContext) {
     );
 }
 
+function handleError(err: any, stream: vscode.ChatResponseStream): void {
+    // making the chat request might fail because
+    // - model does not exist
+    // - user consent not given
+    // - quote limits exceeded
+    if (err instanceof vscode.LanguageModelError) {
```

Comment:
I think it should do something with other errors. Re-throwing is fine, then it will show up in the UI.

Reply:
Good point. Tackled via https://github.com/microsoft/vscode-extension-samples/pull/1041